### PR TITLE
[Security] Add missing Spanish translations

### LINKED ISSUE #41083: Missing translations for Spanish (es)
Hello,

There are some missing translation for Spanish (es). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.es.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.es.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by makraz**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #41083 
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.es.xlf [hunk 1]
```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Enlace de inicio de sesión inválido o expirado.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Demasiados intentos fallidos de inicio de sesión, Inténtelo de nuevo en %minutes% minuto.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Security/Core/Resources/translations/security.es.xlf]: `Inténtelo` -> `inténtelo`

File: src/Symfony/Component/Security/Core/Resources/translations/security.es.xlf [hunk 2]
```diff
@@ -70,6 +70,14 @@
...
+            </trans-unit>
+            <trans-unit id="20">
+                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
+                <target>Demasiados intentos fallidos de inicio de sesión, Inténtelo de nuevo en %minutes% minutos.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Security/Core/Resources/translations/security.es.xlf]: `Inténtelo` -> `inténtelo`

**Review by javiereguiluz — APPROVED** [review]
There's a minor uppercase/lowercase issue to fix, but everything else is great. Thanks @makraz.

**Comment by makraz** [comment]
> There's a minor uppercase/lowercase issue to fix, but everything else is great. Thanks @makraz.

I fixed the typo 
thank you @javiereguiluz

**Comment by derrabus** [comment]
Thank you @makraz.

**Pull request merged**